src/PeriodEnd/Refunds/SFA.DAS.ProviderPayments.Calc.Refunds/Services/Dependencies/ISummariseAccountBalances.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using SFA.DAS.ProviderPayments.Calc.Refunds.Dto;
+using SFA.DAS.ProviderPayments.Calc.Shared.Infrastructure.Data.Entities;
+
+namespace SFA.DAS.ProviderPayments.Calc.Refunds.Services.Dependencies
+{
+    public interface ISummariseAccountBalances
+    {
+        void Initialise();
+        void IncrementAccountLevyBalance(IEnumerable<RefundPaymentEntity> refunds);
+        void IncrementAccountLevyBalance(IEnumerable<AccountLevyCredit> items);
```

Comment:
What's the difference between these two methods? 
Surely only one is required?

Reply:
Actually both methods are required, as the ProviderProcessor is incrementing the balances by refunds Payments and the RefundsProcessor will increment the balances by AccountLevyCredit.

Reply:
Still don't get it. Surely only one of them needs to increment the balances? Otherwise there'll be double credits?

Reply:
No, you'll see why in the subsequent PR